index.html

```diff
@@ -1,6 +1,10 @@
 <!DOCTYPE html>
 <html>
 	<head>
+		<script type="text/javascript" src="//platform.linkedin.com/in.js">
```

Comment:
It is the way it's stated in the documentation: https://developer.linkedin.com/docs/getting-started-js-sdk (in the "Initialize the SDK in your webpage" section).